.github/workflows/test.yml

```diff
@@ -65,3 +64,53 @@ jobs:
         run: |
           cd test
           python3 test.py -vvv
+
+  rhel9_test:
+    name: Unit tests on RHEL9
+    runs-on: redhat/ubi9
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set bazel version to 6.5.0
+        run: echo "6.5.0" > .bazelversion
+
+      - name: Setup Bazel
+        uses: bazel-contrib/setup-bazel@0.15.0
+
+      - name: Install python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.12'
+
+      - name: Install CodeChecker analyzers
+        run: |
+          dnf update -y && \
+          dnf install -y \
+            llvm-toolset \
+            clang-tools-extra \
+            wget \
+            git \
+            python3 \
+            python3-pip \
+            gcc \
+            g++ \
+            scl-utils \
+
+      - name: Install CodeChecker
+        run: pip3 install codechecker
+
+      - name: Print versions
+        run: |
+          bazel version
+          CodeChecker version
+          echo "[NOTE]: If you are debugging, its possible that " \
+               "CodeChecker finds different analyzers when running in " \
+               "bazel's sandbox environment!"
+          CodeChecker analyzers
```

Comment:
Nitpick: why do we need this then?

Reply:
Considering that this, in my experience, one of the more frustrating parts of debugging, I'd be happy if we left in some warning, even if its not pretty. If it even once saves 3 hours of debugging for someone, its worth it. I will concede this isn't the perfect warning message, so lets massage it a bit and keep it.